components/cf-app-push/frontend/src/cf-app-push.module.js

```diff
@@ -0,0 +1,13 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('cf-app-push', [ 'cloud-foundry' ])
+    .run(register);
+
+  /* eslint-disable no-unused-vars */
+  // Ensure that an instance of appDeployAppService is created by injecting it here.
+  function register(appDeployAppService) { }
```

Comment:
Is this needed? Isn't there a danger an optimizer will remove the empty function?